mmv1/products/container/api.yaml

```diff
@@ -207,6 +207,32 @@ objects:
               Whether the nodes are created as preemptible VM instances. See:
               https://cloud.google.com/compute/docs/instances/preemptible for
               more information about preemptible VM instances.
+          - !ruby/object:Api::Type::NestedObject
```

Comment:
technically the changes to api.yaml are not required, since this resource is handwritten - but it also doesn't hurt anything. It just isn't used.

Reply:
Yeah just wanted to be as complete as possible :)